tests/python/frontend/pytorch/test_forward.py

```diff
@@ -1606,6 +1633,39 @@ def verify_script_model(pt_model, ishapes):
                                     rtol=1e-5, atol=1e-5)
 
 
+def verify_trace_model(traced_model, ishapes, idata=None):
```

Comment:
This new function you added is almost identical with `verfiy_script_model` https://github.com/apache/incubator-tvm/blob/master/tests/python/frontend/pytorch/test_forward.py#L1578

Please introduce another function, verify_model_on_vm say, to refactor the common logic of vm execution and verifying

Reply:
@masahi did you mean removing `verify_trace_model` and adding `verify_model_on_vm` to serve as a common function for vm execution?

Reply:
I didn't mean to say remove this particular function, but yeah, since `verify_model` used by most of other tests are already meant to be used for traced model, having another `verify_trace_model` doesn't seem right.

Running traced models on VM is a new test case that we haven't had so far, and there are already some tests (for models jitted by `torch.jit.script`) that use VM execution,  we need a new function that handles

* Traced + VM (new in this PR)
* Scripted + VM

Reply:
Then plan to introduce verify_model_on_vm to work for both traced and scripted, and remove verify_trace_model & verify_script_model, does it make sense?

Reply:
yes makes sense, but what I'm imagining is that there would be still `verify_script_model` and the existing tests that use it shouldn't be touched. `verify_script_model` would become a simple wrapper around `verify_model_on_vm`.

`verify_model_on_vm` should take already jitted models as input, which can be both traced or scripted. You shouldn't have `torch.jit.trace` or `torch.jit.script`. You probably want another wrapper to handle trace + VM case.